commands/inventory/list_agents.go

```diff
@@ -19,12 +19,29 @@ import (
 	"strings"
 
 	"github.com/AlekSi/pointer"
+	"github.com/percona/pmm/api/inventorypb"
 	"github.com/percona/pmm/api/inventorypb/json/client"
 	"github.com/percona/pmm/api/inventorypb/json/client/agents"
 
 	"github.com/percona/pmm-admin/commands"
 )
 
+const (
```

Comment:
let's move it to pmm repo and figure out how to not import protobuf and grpc packages